ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -301,8 +503,9 @@ private string GetCursorName(Decl decl)
 
                 default:
                 {
-                    Unhandled(decl);
-                    return string.Empty;
+                    var name = decl.Spelling;
+                    AddDiagnostic(DiagnosticLevel.Warning, $"Unsupported declaration: '{decl.KindSpelling}'. Falling back to '{name}'.", decl);
```

Comment:
Things where the binding generation support is incomplete, but where we have a "reasonable" fallback are currently warnings. They generally represent cases where we might generate the correct code but it should really be investigated/handled properly.